apps/rpc/src/modules/event/attendance-router.ts

```diff
@@ -7,26 +7,23 @@ import {
   AttendeeSelectionResponsesSchema,
   UserSchema,
 } from "@dotkomonline/types"
+import { TRPCError } from "@trpc/server"
 import { z } from "zod"
 import { authenticatedProcedure, procedure, staffProcedure, t } from "../../trpc"
 
 export const attendanceRouter = t.router({
-  getAttendee: authenticatedProcedure
+  createPool: staffProcedure
     .input(
       z.object({
-        userId: z.string(),
-        attendanceId: AttendanceSchema.shape.id,
+        id: AttendanceSchema.shape.id,
```

Comment:
```suggestion
        id: AttendancePoolSchema.shape.id,
```
This can be confusing since the prop is named just `id` and it is the wrong type

Reply:
Unless this is supposed to be like this.
I still think we should long-term switch to using like `attendanceId: AttendanceSchema.shape.id` over just `id: Something.shape.id` so it's straight forward, but this pr is big enough